libkbfs/folder_branch_ops.go

```diff
@@ -3576,21 +3577,35 @@ func (fbo *folderBranchOps) SetMtime(
 		})
 }
 
-func (fbo *folderBranchOps) syncLocked(ctx context.Context,
-	lState *lockState, file path) (stillDirty bool, err error) {
+// startSyncLocked readies the blocks and other state needed to sync a
+// single file.  It returns:
+//
+// * `doSync`: Whether or not the sync should actually happen.
```

Comment:
3/4 are possible.  `doSync=true, stillDirty=false` doesn't make sense because there would be nothing to sync.  @akalin-keybase do you want it to be something other than two bools?